README.md

```diff
@@ -68,6 +68,7 @@ myChart
 | <b>focusOnNode</b>([<i>object</i>]) | Getter/setter for the data node to focus the chart on. Use this method to retrieve the current node in focus, or to programmatically zoom the chart to a particular node. | |
 | <b>onHover</b>([<i>fn</i>]) | Callback function for mouse hover events. Includes the data node object (or `null` if hovering on background) as single argument. | |
 | <b>onClick</b>([<i>fn</i>]) | Callback function for click events. Includes the data node object (or `null` if clicking on background) as single argument. A falsy value (default) automatically focuses on clicked nodes, equivalent to `myChart.onClick(myChart.focusOnNode)`. | |
+| <b>animationDuration</b>([<i>number</i>]) | Set the animation duration when the chart appears on screen. Enter 0 to disable the animation. The default is `750` (ms). | `750` |
```

Comment:
As mentioned in the other comment, this value affects every transition, not just the initial one. Could you please update the description?